agcaballero/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -73,15 +76,29 @@ public String toString() {
     /**
      * @return a Comment as a HTML div with proper formatting to be displayed
      */
-    private String htmlFormat() {
+    private String htmlFormat(String languageCode) {
       // gets time in local time zone (default: US ET)
       LocalDateTime localTime =
           new LocalDateTime(timestamp.getTime(), DateTimeZone.forID("US/Eastern"));
       DateTimeFormatter formatter = DateTimeFormat.forPattern("h:mm a M/dd/yy");
 
       return "<div class='comment-div'>"
           + "<p class='date'>" + formatter.print(localTime) + "</p>"
-          + "<p><b>" + name + " (" + email + "):</b> <br><br>" + text + "</p></div>";
+          + "<p><b>" + name + " (" + email + "):</b></p>"
+          + "<p class='comment-text' lang=" + languageCode + ">" + text + "</p></div>";
+    }
+
+    /**
+     * Translate the text to the language of a given language code
```

Comment:
I will add to the comment to say (e.g. "EN" for English). @ykjchen do you think this is sufficient clarification?

Reply:
Yeah, that helps to know it's a 2 character language code. Ideally, you'd make note of the ISO spec (if it uses one), so that users know what format to pass in here, but for the purposes of this exercise, if that's not readily available, your example would be sufficient.

Reply:
I'll try and provide the language spec if I can find it.

Reply:
Done (I found the ISO spec)